src/popups/HandicapPopup/HandicapPopup.scss

```diff
@@ -6,6 +6,18 @@
     text-decoration: underline;
   }
 
+  a.wkp-link {
+    text-decoration: none !important;
+  }
+
+  .wkp-link span {
+    text-decoration: underline;
+    text-overflow: ellipsis;
+    max-width: 380px;
+    overflow: hidden;
+    display: inline-block;
```

Comment:
With `display: inline-block;` the barrierfree focus box isn't perfect as the height at the arrow is smaller than at the text.
`display: table-cell;` could solve it. But it is [not supported by Opera Mobile](https://caniuse.com/#search=table-cell). If we don't want to ignore Opera Mobile, we could additionally add `display: table-cell;` after `display: inline-block;`. Then, I think, all browser would use `display: table-cell;` except for Opera Mobile. @edellucien what do you think?

Reply:
Set the arrow svg height, the same height as the text to have a correct highlight.
eg. 16px

Reply:
Solved with the suggestion of https://github.com/geops/trafimage-maps/pull/676#discussion_r422097480